posthog/api/action.py

```diff
@@ -184,10 +184,15 @@ def list(self, request: request.Request, *args: Any, **kwargs: Any) -> Response:
         actions_list: List[Dict[Any, Any]] = self.serializer_class(actions, many=True, context={"request": request}).data  # type: ignore
         return Response({"results": actions_list})
 
+    # NOTE: Deprecated in favour of `persons/trends` endpoint
+    # Once the old way of exporting CSVs is removed, this endpoint can be removed
     @action(methods=["GET"], detail=False)
     def people(self, request: request.Request, *args: Any, **kwargs: Any) -> Response:
         team = self.team
         filter = Filter(request=request, team=self.team)
+        if not filter.limit:
```

Comment:
this seems wrong, my fix for trends for person-on-events that went in a day or two ago should've gotten rid of the `LIMIT 200`

(apologies about the merge conflict)

Reply:
My goal here: It should be 100, not 200 😅

Reply:
Gotcha 👍